pandas/core/shared_docs.py

```diff
@@ -75,6 +75,11 @@
 keep_equal : bool, default False
     If true, the result keeps values that are equal.
     Otherwise, equal values are shown as NaNs.
+
+suffixes : tuple, default ('self', 'other')
```

Comment:
i think we should call this `names`  or maybe `result_names` as suffixes implies we are *appending* these to something (ala merge)

Reply:
@jreback Sorry, I didn't see the comment.
I guess suffix should be used because to some extent the result will be merged. What I mean is that the comparison will be one block representing the different parts of the two dataframes.

Reply:
I can see your point but result names will mean that the result will have names. like we have an output which will have multiple names

Reply:
@jreback I edited the name of the argument